packages/elements/src/components/ino-dialog/ino-dialog.tsx

```diff
@@ -115,6 +144,11 @@ export class Dialog implements ComponentInterface {
   }
 
   render() {
+  
+
+    //const hasBodySlot = hasSlotContent(this.el, 'body');
+    const hasContentSlot = hasSlotContent(this.el, 'content');
```

Comment:
Instead of using one slot for all, I would suggest using three slots `header`, `footer`, and `content`. That way, we allow the consumer to change just one thing and still use our design for the rest.

Pseudocode:
```
hasHeaderContent && {<header...>}
hasBodyContent && {<div....>}
hasFooterContent && {<footer...>}
```

But I would also allow the user the replace everything by providing elements to the default slot.

Pseudocode:

```
hasDefaultContent ? <slot></slot> : {
....
} 
```